pkgs/by-name/ay/ayatana-indicator-messages/package.nix

```diff
@@ -40,6 +40,10 @@ stdenv.mkDerivation (finalAttrs: {
     "dev"
   ] ++ lib.optionals withDocumentation [ "devdoc" ];
 
+  patches = [
+    ./fix-pie.patch
```

Comment:
Any reason to not just `fetchpatch` it?

Reply:
Yes. It is not yet merged, so it could disappear if you were to force-push to the PR branch.

Reply:
Alrighty, section got added since the last time I had a deep look at `pkgs/CENTRIBUTING.md` that properly details when & why to do this: https://github.com/NixOS/nixpkgs/blob/2c8f8a719a9018a43984062272ebf5772dc786d6/pkgs/README.md#vendoring-patches

Are we in a rush to get this resolved? Otherwise I'd just wait abit for upstream to respond to the PR, to see if we can avoid adding the patch to Nixpkgs.

Reply:
There is a chance PIE will be enabled for the next staging cycle. In that case we have about 5-6 weeks.